packages/flutter_tools/test/general.shard/dart_plugin_test.dart

```diff
@@ -545,22 +536,27 @@ void main() {
             ),
           ]);
 
-          expect(
-            testLogger.errorText,
-            'Plugin `url_launcher_linux_2` implements an interface for `linux`, which was already implemented by plugin `url_launcher_linux_1`.\n'
-            'To fix this issue, remove either dependency from pubspec.yaml.'
-            '\n\n'
-          );
         },
         throwsToolExit(
           message: 'Please resolve the errors',
         ));
+
+        expect(
```

Comment:
These error text expectations were moved because they didn't actually do anything where they were; they were at the end of a try/catch that was expected to throw.